packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -94,20 +99,26 @@ export const DropdownContent = (
     {
         side = 'bottom',
         align = 'start',
+        distance = 'compact',
         children,
         preventTriggerFocusOnClose,
         'data-test-id': dataTestId = 'fondue-dropdown-content',
     }: DropdownContentProps,
     ref: ForwardedRef<HTMLDivElement>,
 ) => {
     const theme = useFondueTheme();
+    const spacingMap = {
+        compact: 8,
+        spacious: 12,
+    };
```

Comment:
Extract out of the component
```suggestion
    const SPACING_MAP: Record<DropdownContentProps['padding'], number> = {
        compact: 8,
        spacious: 12,
    };
    
    export const Dropdown // ....
```